apps/condo/domains/acquiring/constants/context.js

```diff
@@ -10,10 +10,20 @@ const CONTEXT_STATUSES = [
     CONTEXT_FINISHED_STATUS,
 ]
 
+const VAT_OPTIONS = [0, 10, 20]
```

Comment:
What's this? 
I guess it's country specific and should be provided by specific Acquiring integration...

Reply:
This is an array with options for `Select`. All VAT values are already included in payment sum. These values need to be printed on the operation receipt.

Reply:
Is it country specific field?

Reply:
No. These are possible values of VAT for MVP. If we decide to split these values by country we will do it.
0, 10, and 20 are values for RU.